### DISCUSSION TITLE
Default text for project-gc.com - DiscoverTrackables

### DISCUSSION THREAD

**Discussion by ChristianGK-GC** [2Abendsegler/GClh#2151] [Ideas]
I'd implement a default text for https://project-gc.com/Tools/DiscoverTrackables
Similar function like "Cache log signature?" in GCLH.
Shall I implement it in GCLH or keep it seperate?

**Comment by 2Abendsegler** [comment]
Moin,

der GClh bietet die Möglichkeit eine [TB log signature](https://www.geocaching.com/my/#GClhShowConfig#a#settings_tb_signature) zu hinterlegen. Ich nutze das als Default Text für ein TB Log. Theoretisch könnte man diesen Text auch auf der Project-GC Seite nutzen. 

Aber wäre nicht das Script von Project-GC der bessere Platz um das Thema abzubilden? Bei dem Thema passiert ja auch gar nichts mehr auf den Geocaching Webseiten.

**Comment by ChristianGK-GC** [comment]
Moin moin,

vermutlich wäre es besser, aber wie du schon sagst existiert die Funktion Teilweise schon in GCLH.
Wie man aber sieht, finden schon sehr viele Änderungen auf den GC Seiten statt.
![grafik](https://user-images.githubusercontent.com/24353480/176829504-aec6b67a-b852-43e0-bbf6-37cbbe850936.png)

**Comment by 2Abendsegler** [comment]
Ich meinte, dass der GClh die Geocaching Webseiten beackert und nicht die Project-GC Webseiten. Wir gehen für gewöhnlich nur dann auf andere Seiten, wenn wir damit etwas auf den GC Seiten realisieren möchten. So wie beispielsweise das automatische Hinzufügen der Pocket Queries aus den Daten die nur Project-GC zur Verfügung stellt. Oder wenn wir auf anderen Seiten einen Link zu den GC Karten setzen möchten, wie wir das bei Goggle Maps oder Openstreetmap machen.  

Hier geht es jedoch darum eine Funktionalität auf den Project-GC Seiten auszuweiten bzw. zu implementieren. Speziell bei Project-GC kommt noch hinzu, dass es dort unter anderem genau für solche Dinge bereits ein entsprechendes Script gibt und wir mit der geplanten Änderung auf der Spielwiese dieses Scriptes rumspielen würden. Wenn es irgendwie geht möchten wir nicht in Konkurrenz zu anderen Skripten treten, sondern viel mehr mit deren Betreibern zusammen arbeiten. 

Ich weiß nicht so recht, ob wir diese Entwicklung im GClh aufnehmen sollen? Vielleicht übersehe ich ja noch etwas, aber spricht noch etwas dagegen es im Project-GC Script zu implementieren?

**Reply by ChristianGK-GC** [reply]
Ich denke nicht, dass du was übersiehst.
Der Gedanke war für mich nur, dass bereits ein Wert für den TB Log schon in GCLH ist.

**Comment by capoaira** [comment]
Moin,
das Script von PGC integriert hauptsächlich Funktionen von PGC auf geocaching.com und räumt die Seiten etwas auf.
Auf deren eigenen Seiten verändert das Script nichts. Das würde aber auch wenig Sinn machen, da sie das ja auch selber in ihrem Web-Quellcode anpassen könnten.
Trotzdem bin ich bei @2Abendsegler. Der GClh hat ja den Slogan "Some little things to make life easy (on **geocaching.com**)"😉
Wenn wir jetzt noch auf anderen Seiten aktiv werden, wird der GClh noch aufgeblähter. Selbst ich als Entwickler habe teils Schwierigkeiten Funktionen im config zu finden, obwohl ich es eigentlich auswendig kennen müsste 😅
Und das Bug Potenzial steigt noch weiter...

Ich sehe aber auch nicht das Problem, man kann bis zu 1000 TBs auf einmal über PGC loggen, dafür jedes mal einen neuen Text anzulegen macht wohl nicht so viel Aufwand. (Da man meiner Meinung nach mindestens den Ort/das Event angeben sollte, wo man den TB gesehen hat, müsste man so oder so mit jedem neuen Discovern den Text abändern.)

**Comment by ChristianGK-GC** [comment]
Alles klar, dann weg mit dem Vorschlag.
